core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.html

```diff
@@ -84,31 +94,23 @@
     font-size: 1.7em;
   }
 
-  audio-bar md-progress-linear.md-default-theme .md-bar {
+  .mat-progress-bar.mat-primary {
     background-color: #009688;
   }
```

Comment:
Why do we need this change, I think the primary color is already defined to this color, right?

Reply:
verified and removed.